doc/classes/Vector2.xml

```diff
@@ -43,7 +43,7 @@
 			<return type="float">
 			</return>
 			<description>
-				Returns this vector's angle with respect to the X axis, or [code](1, 0)[/code] vector, in radians.
+				Returns this vector's angle with respect to the [code](1, 0)[/code] vector (i.e. [constant Vector2.RIGHT] has the angle of 0), in radians. The result is always positive.
```

Comment:
I think it would be better to say "The result is always greater than 0." or "The result is always greater than or equal to 0.".

This way, we can know whether the value can be equal to 0 or not. Even if it might be obvious from the description, it doesn't hurt to clarify it :slightly_smiling_face:

Reply:
0 is not a positive number, so it should probably be non-negative?